### ISSUE TITLE
[Bug] Failed to get refElement.

### ISSUE THREAD

**Issue by johncodeai** [Darginec05/Yoopta-Editor#290]
### Has this bug been raised before?

- [X] I have checked "open" AND "closed" issues and this is not a duplicate

### Description

```javascript
function WithBaseFullSetup() {  
  const editor = useMemo(() => createYooptaEditor(), []);

  useEffect(() => {
    if(!editor.refElement) return 
    // Trying to get the editor element from the instance fails, this will never be executed
    console.log('output:', editor.refElement)
  }, [editor.refElement])

  return (
    <YooptaEditor
      editor={editor}
      // ...other props
    />
  );
}

```
The problem: Trying to get the editor element from the instance fails, `console.log('output:', editor.refElement)` will never be executed.

### Steps to Reproduce

This problem occurs when trying to access editor.refElement

### Environment

- OS: MacOs

### Do you want to work on this issue?

No

**Comment by Darginec05** [comment]
I published `-rc` version with this fix.
Try it :)

**Comment by Darginec05** [comment]
Fixed in v4.8.0
